### PR TITLE
Update the example

### PR DESCRIPTION
**Pull request by yuri1969**
The previous example syntax was broken and used pre-2021 standards.

### CODE REVIEW

**Comment by loicmathieu** [comment]
Thanks, we'll need to check it and update also the `TeamsExecution` task.

**Comment by yuri1969** [comment]
@loicmathieu The doc for the rest of the tasks is updated now.

However, the `​Teams​Execution` doc still mentions Slack since the task inherits properties from `SlackIncomingWebhook`.

**Comment by tchiotludo** [comment]
@yuri1969 seems a little bit dirty that teams inherit from slack, I think we should recreate a proper task to avoid this heritage

**Comment by yuri1969** [comment]
@tchiotludo That Slack reference was likely a copy&pate error.

**Comment by loicmathieu** [comment]
@yuri1969 

What I said by

> we'll need to check it and update also the TeamsExecution task.

Is that we also need to update the  TeamsExecution template which is here: https://github.com/kestra-io/plugin-notifications/blob/master/src/main/resources/teams-template.peb with the new format.

Can you have a look ?
If you don't have time I'll do, no problem ;)

**Comment by yuri1969** [comment]
@loicmathieu That template seems to be fine - no links in `facts`.

File: src/main/java/io/kestra/plugin/notifications/teams/TeamsExecution.java [hunk 1]
```diff
@@ -43,11 +42,12 @@
                 "    tasks:",
                 "      - id: teams",
                 "        type: io.kestra.plugin.notifications.teams.TeamsExecution",
+                "        url: \"https://TENANT-NAME.webhook.office.com/webhookb2/XXXXXXXXXX\"",
```
Comment (loicmathieu) [inline src/main/java/io/kestra/plugin/notifications/teams/TeamsExecution.java:45]:
```suggestion
                "        url: \"https://microsoft.webhook.office.com/webhookb2/XXXXXXXXXX\"",
```

File: src/main/java/io/kestra/plugin/notifications/teams/TeamsIncomingWebhook.java [hunk 2]
```diff
@@ -33,22 +33,30 @@
         @Example(
             title = "Send a Microsoft Teams notification message",
             code = {
-                "url: \"https://microsoft.webhook.office.com/webhook/XXXXXXXXXX\"",
+                "url: \"https://TENANT-NAME.webhook.office.com/webhookb2/XXXXXXXXXX\"",
```
Comment (loicmathieu) [inline src/main/java/io/kestra/plugin/notifications/teams/TeamsIncomingWebhook.java:36]:
```suggestion
                "url: \"https://microsoft.webhook.office.com/webhookb2/XXXXXXXXXX\"",
```

**Review by loicmathieu — CHANGES_REQUESTED** [review]
I wonder if `url` shouldn't be required.

File: src/main/java/io/kestra/plugin/notifications/teams/TeamsExecution.java (hunk 1 above)
Comment (yuri1969) [reply src/main/java/io/kestra/plugin/notifications/teams/TeamsExecution.java:45]: The lowest domain level is actually a Microsoft 365 tenant. Usually, the value is equal to the _company domain name_. Thus presenting the `microsoft` value there might IMHO be confusing a bit.

**Comment by yuri1969** [comment]
@loicmathieu The `url` property is now mandatory.

File: src/main/java/io/kestra/plugin/notifications/teams/TeamsIncomingWebhook.java [hunk 3]
```diff
@@ -59,6 +69,8 @@ public class TeamsIncomingWebhook  extends Task implements RunnableTask<VoidOutp
         title = "Microsoft Teams incoming webhook URL"
     )
     @PluginProperty(dynamic = true)
+    @NotNull
+    @NotEmpty
```
Comment (loicmathieu) [inline src/main/java/io/kestra/plugin/notifications/teams/TeamsIncomingWebhook.java:73]: `@NotEmtpy` is enought as it also means not-null.

**Review by loicmathieu — CHANGES_REQUESTED** [review]
Can you remove all the invalid flows that you added, this was a good idea but such tests (that validation annotations works on a flow definition) are already covered by test in Kestra core so no need to add them everywhere.

File: src/main/java/io/kestra/plugin/notifications/teams/TeamsExecution.java (hunk 1 above)
Comment (loicmathieu) [reply src/main/java/io/kestra/plugin/notifications/teams/TeamsExecution.java:45]: Agreed but we want example to be as close as copy/paste working as possible, I think Microsoft 365 will be used a lot these days so I think it's good to have it as an example.

**Comment by yuri1969** [comment]
> Can you remove all the invalid flows that you added, this was a good idea but such tests (that validation annotations works on a flow definition) are already covered by test in Kestra core so no need to add them everywhere.

Done

**Comment by loicmathieu** [comment]
Thanks a lot @yuri1969 it's OK for me, I'll merge it as soon as CI is green.

**Review by loicmathieu — APPROVED** [review]
LGTM, thanks

**Pull request merged**